src/Dependencies/SourcePackage.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.cs]
+generated_code = true
```

Comment:
For the clasp source package, I specifically had issues with compiler documentation warnings as well
https://github.com/dotnet/roslyn/blob/main/src/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/contentFiles/.editorconfig#L14C19-L14C25
Not sure if that should be included, or if there is a better way to disable it